File: 0088-merge-sorted-array/0088-merge-sorted-array.py
class Solution:
    def merge(self, nums1: List[int], m: int, nums2: List[int], n: int) -> None:
        """
        Do not return anything, modify nums1 in-place instead.
        """
        i = 0
        while i < m:
            j=0
            while j < n:
                if nums1[i] >= nums2[j]:
                    nums1.insert(i,nums2.pop(j))
                    nums1.pop()
                    m += 1
                    n -= 1
                j += 1
            i += 1   
        while n > 0:
            nums1.insert(i,nums2.pop(0))
            nums1.pop()
            i += 1
            n -= 1

File: 0088-merge-sorted-array/0088-merge-sorted-array.c
void merge(int* nums1, int nums1Size, int m, int* nums2, int nums2Size, int n){
        int i=0, j=0, k=0;
        int nums3[n+m];
        memcpy(nums3, nums1, (n+m)*sizeof(int));
        do {

            if(j == n || (nums3[i] < nums2[j] && i < m)){
                nums1[k++] = nums3[i++];
            }
            else{ 
                nums1[k++] = nums2[j++];
            }
        } while (k < n+m);
}
